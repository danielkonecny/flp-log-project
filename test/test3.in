A B
A B
B C
C D
D A
