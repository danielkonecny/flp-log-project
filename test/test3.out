A-B A-D B-C
A-B A-D C-D
A-B B-C C-D
A-D B-C C-D
